README.md

```diff
@@ -17,7 +17,7 @@ Eiffel Goer implements the event repository API and is intended as an open sourc
 
 ### Docker
 
-    docker run -e CONNECTION_STRING=yourdb -e DATABASE_NAME=dbname -e API_PORT=8080 registry.nordix.org/eiffel/goer
+    docker run -e CONNECTION_STRING=yourdb -e API_PORT=8080 registry.nordix.org/eiffel/goer
```

Comment:
In the makefile we always push to a tag that matches `git describe`. Will we ever have a latest tag that would be matched by this command?

Reply:
I will leave it as is for now, but I see three scenarios that we could follow here:

1. Always push latest when we push a new tag.
2. Add an `export VERSION=git describe` command in the README.
3. Always update the README when doing new version tags.

The third option feels best, but I will forget to do it on releases :)